corehq/apps/app_manager/models.py

```diff
@@ -843,6 +843,27 @@ def check_paths(self, paths):
 
         return errors
 
+    def add_property_save(self, app_case_meta, case_type, name,
+                          questions, question_path, condition=None):
+        if question_path in questions:
+            app_case_meta.add_property_save(
+                case_type,
+                name,
+                self.unique_id,
+                questions[question_path],
+                condition
+            )
+
+    def add_property_load(self, app_case_meta, case_type, name,
+                          questions, question_path):
+        if question_path in questions:
+            app_case_meta.add_property_load(
+                case_type,
+                name,
+                self.unique_id,
+                questions[question_path]
+            )
```

Comment:
If the question doesn't exist you should call `app_case_meta.add_property_error` so that the user is aware of the error.